aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +31,105 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
+            include different actions a user can perform. Icons can represent
+            different types of functionality. It is important that the context
+            of an icon is clear. In some cases, it may be beneficial to pair
+            icons with text to ensure the purpose of the icon is clear.
+          </SubsectionText>
+        </Subsection>
+        <ButtonRow>
+          <Button
+            label="See in Figma"
+            href="https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons"
+            primary
+            target="_blank"
+            rel="noreferrer noopener"
+          />
+        </ButtonRow>
+      </ContentSection>
```

Comment:
Still don't think we should separate each subsection in their own ContentSection. I think we can put them all in one ContentSection.

Reply:
okay I think I missed this the first time around